omod/src/main/webapp/fragments/patientHeader.gsp

```diff
@@ -119,27 +119,18 @@
                     <i class="toggle-icon icon-caret-up small"></i>
                 </a>
             </span>
+
+            <div class="firstLineFragments">
+                <% firstLineFragments.each { %>
+                    ${ ui.includeFragment(it.extensionParams.provider, it.extensionParams.fragment, [patient: config.patient])}
+                <% } %>
+            </div>
+
             <div class="hidden" id="contactInfoContent" class="contact-info-content">
                 ${ ui.includeFragment("coreapps", "patientdashboard/contactInfoInline", [ patient: config.patient, contextModel: appContextModel ]) }
             </div>
         </h1>
-        <% if (config.activeVisit) { %>
-            <% def visit = config.activeVisit.visit %>
 
-            <div class="active-visit-started-at-message">
-                ${ui.message("coreapps.patientHeader.activeVisit.at", config.activeVisitStartDatetime)}
```

Comment:
Wouldn't the fragments be loaded here, in the block that you are replacing, instead of line 187?